components/applied-filters/applied-filters.js

```diff
@@ -66,11 +64,6 @@ class D2lLabsAppliedFilters extends RtlMixin(FilterLocalizeMixin(LitElement)) {
 				padding-top: 0.3rem;
 			}
 
```

Comment:
Not part of your change, but it looks like `.d2l-labs-applied-filters-label` can go away too, and it was the only thing requiring the `RtlMixin`.